src/index.js

```diff
@@ -1,96 +1,55 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
```

Comment:
we need the following modules
cors, session, passport

Reply:
Mongoose too  ` mongoose = require("mongoose");`

Reply:
No, we don't need mongoose, we are going to use sequalize